src/patternfly/components/Sidebar/sidebar.scss

```diff
@@ -88,6 +88,15 @@ $pf-c-sidebar__panel--list--width: ("default", 25, 33, 50, 66, 75, 100);
     }
   }
 
+  &.pf-m-full-height {
+    --pf-c-sidebar__main--AlignItems: stretch;
```

Comment:
I'm not sure if this is the var we want to change. Currently since it comes before `.pf-m-split`, it's getting overridden because the split layout sets this, too, so it can switch from stretch to start at a medium breakpoint.

I wonder if `pf-m-full-height` might make more sense on the panel and content, so you can make one or the other full height? It's worth noting that when the panel is full height, it is longer sticky, so maybe it makes sense to be able to make the content full height for whatever reason, so the panel can still be sticky? Or this could come after the stack and split blocks, and maybe just set `--pf-c-sidebar--m-split__main--AlignItems: stretch` since that's what I believe this is targeting.